monitoring/rid_qualifier/host/routes.py

```diff
@@ -47,8 +46,12 @@ def info():
 def login_required(function):
     @wraps(function)
     def decorated_function(*args, **kwargs):
-        if 'google_id' not in session:
+        if flow and 'google_id' not in session:
             return redirect(url_for('login', next=request.url))
+        elif 'google_id' not in session:
+            session['google_id'] = 'testuser'
```

Comment:
Let's just use `'localuser'` since an individual developer may want to run rid_qualifier host on their own local machine to do real (not just test) development (and likewise, `'Local User'` for `session['name']`; I'm not sure what `session['state']` should be)